test/CMakeLists.txt

```diff
@@ -8,31 +8,40 @@ add_unit_test(
     concurrency)
 
 add_unit_test(
-    freestanding_injected_policy_test
+    freestanding_conc_injected_policy_test
     CATCH2
     FILES
-    freestanding_injected_policy.cpp
+    freestanding_conc_injected_policy.cpp
     LIBRARIES
     warnings
     concurrency)
 
 add_unit_test(
-    hosted_injected_policy_test
+    hosted_conc_injected_policy_test
     CATCH2
     FILES
-    hosted_injected_policy.cpp
+    hosted_conc_injected_policy.cpp
     LIBRARIES
     warnings
     concurrency)
 
 add_unit_test(
-    standard_policy_test
+    conc_standard_policy_test
     CATCH2
     FILES
-    standard_policy.cpp
+    conc_standard_policy.cpp
     LIBRARIES
     warnings
     concurrency
     pthread)
 
-add_compile_fail_test(fail_no_policy.cpp LIBRARIES concurrency)
+add_compile_fail_test(fail_no_conc_policy.cpp LIBRARIES concurrency)
+
+add_unit_test(
+    atomic_standard_policy_test
+    CATCH2
+    FILES
+    atomic_standard_policy.cpp
+    LIBRARIES
+    warnings
+    concurrency)
```

Comment:
question: why not add `sanitize` here?

Reply:
It's already added by `add_unit_test` based on the SANITIZERS environment variable.